clickhouse-mixin/config.libsonnet

```
@@ -1,20 +1,26 @@
 {
-  _config+:: {
-    enableMultiCluster: false,
-    clickhouseSelector: if self.enableMultiCluster then 'job=~"$job", instance=~"$instance", cluster=~"$cluster"' else 'job=~"$job", instance=~"$instance"',
+  enableMultiCluster: false,
```

```suggestion
  enableMultiCluster: false,
```

overall, I think cluster could be considered a specific case of a member of groupLabel. Do we need any specific support for clusterLabel?

---

I'm curious about this as well. In this effort I'm mostly trying to refactor over to the new formats in the least destructive way. I've noticed that for like velero-2 we've not had this configuration field in config.libsonnet, but for others like pgbouncer we do something similar...

I'm not quite sure what the ideal preference is here, but the `enableMultiCluster` seems to at least make initial sense to me but happy to explore what is the preferred approach!

---

Apologies as its been a while since fully completing one of these, but let me know what the ideal for this would be, it seems like we should keep the config enableMultiCluster in my opinion but I may be missing something.

If there's a good example or guidance I can go off of I'm happy to make further changes! Otherwise I believe this is probably what we'd like?

---

I think cluster label is just yet another groupLabel. It is only unique in a way that it could be missing in metrics, so allValue filter should be `.*` not `.+` as others. Also this is more about integrations context and not really about mixin.

@Dasomeone , what do you think? I personal lean towards removing it everywere and finding a generic way to set some of the groupLabels "optional".